bootstrap/lib/util.js

```diff
@@ -8,6 +8,15 @@ const parseBool = (value) => {
   return (_val === 'true' || _val === 'false') && _val === 'true'
 }
 
+// parse incoming query string, strings into Numbers where possible
+const parseQueryString = (query) => {
+  let formatted = {}
+  for (const [key, val] of Object.entries(query)) {
+    formatted[key] = isNaN(val) ? val : Number(val)
+  }
+  return formatted
```

Comment:
Perfect opportunity to use ES2019 `Object.fromEntries`:

```suggestion
  const toNumber = (v) => isNaN(v) ? v : Number(v)
  return Object.fromEntries(Object.entries(obj).map(([k, v]) => [k, toNumber(v)]))
```